src/hosted-buttons/utils.test.js

```diff
@@ -288,4 +292,81 @@ describe("buildHostedButtonOnApprove", () => {
   });
 });
 
+test("getFlexDirection", () => {
+  expect(getFlexDirection({ layout: "horizontal" })).toStrictEqual({
+    flexDirection: "row",
+  });
+  expect(getFlexDirection({ layout: "vertical" })).toStrictEqual({
+    flexDirection: "column",
+  });
+});
+
+test("getButtonColor", () => {
+  const colors = ["gold", "blue", "silver", "white", "black"];
+  const fundingSources = ["paypal", "venmo", "paylater"];
+  const colorMap = {
+    gold: {
+      paypal: "gold",
+      venmo: "blue",
+      paylater: "gold",
+    },
+    blue: {
+      paypal: "blue",
+      venmo: "silver",
+      paylater: "blue",
+    },
+    black: {
+      paypal: "black",
+      venmo: "black",
+      paylater: "black",
+    },
+    white: {
+      paypal: "white",
+      venmo: "white",
+      paylater: "white",
+    },
+    silver: {
+      paypal: "silver",
+      venmo: "blue",
+      paylater: "silver",
+    },
+  };
+
+  colors.forEach((color) => {
+    fundingSources.forEach((fundingSource) => {
+      expect(getButtonColor(color, fundingSource)).toBe(
+        colorMap[color][fundingSource]
+      );
+    });
+  });
+});
+
+test("shouldRenderSDKButtons", () => {
+  expect(shouldRenderSDKButtons([])).toBe(false);
+  expect(shouldRenderSDKButtons(["paypal"])).toBe(true);
+  expect(shouldRenderSDKButtons(["paypal", "venmo"])).toBe(true);
```

Comment:
optional naming suggestion: `shouldRenderStandaloneButtons`

Reply:
I called it `shouldRenderSDKButtons` thinking we could reuse it and update the logic when we get the `checkoutButtonType` variable from the updated response.

We wont want to run any of the code in that if statement when `checkoutButtonType=single`

![Screenshot 2024-04-17 at 11 44 45 AM](https://github.com/paypal/paypal-checkout-components/assets/32575209/4ee0993d-ae5a-4624-9929-6717e08df093)